TCMSApp/src/client/app/runs/runs-exec/runs-exec.route.js

```diff
@@ -16,14 +16,14 @@
             {
                 state: 'runs-edit',
                 config: {
-                    url: '/runs/edit',
+                    url: '/runs/:id/edit',
                     templateUrl: 'app/runs/runs-exec/runs-edit.html',
                     controller: 'RunsEditController',
                     controllerAs: 'vmRunsEdit',
                     title: 'Edit Test Run',
                     params: {
                         run: {
-                            value: {},
+                            value: undefined,
                             squash: false
```

Comment:
squash - {bool|string=}: squash configures how a default parameter value is represented in the URL when the current parameter value is the same as the default value. If squash is not set, it uses the configured default squash policy. (See defaultSquashPolicy())

There are three squash settings:

false: The parameter's default value is not squashed. It is encoded and included in the URL
true: The parameter's default value is omitted from the URL. If the parameter is preceeded and followed by slashes in the state's url declaration, then one of those slashes are omitted. This can allow for cleaner looking URLs.
"<arbitrary string>": The parameter's default value is replaced with an arbitrary placeholder of your choice.